jujugui/static/gui/src/app/app.js

```diff
@@ -1187,6 +1196,11 @@ YUI.add('juju-gui', function(Y) {
           showPay={window.juju_config.payFlag || false}
           updateCloudCredential={
             controllerAPI.updateCloudCredential.bind(controllerAPI)}
+          updateAddress={
```

Comment:
I wonder if we should just provide `this.payment` to the components, rather than 6 methods.
Something like `paymentlib={this.payment}`